Dockerfile

```
@@ -15,6 +15,12 @@ RUN ( [ ! -e "/usr/local/share/ipa-container" ] \
          || rm -f /usr/local/sbin/init ) \
     && ln -svf /usr/local/share/ipa-container/init.sh /usr/local/sbin/init
 COPY ./init /usr/local/share/ipa-container
+COPY ./tmpfiles.conf /usr/lib/tmpfiles.d/00-ipa-container.conf
+
+# Completely replace systemd-tmpfiles.  This is needed until FreeIPA
+# itself provides a way to select the tmpfiles implementation via
+# the 'ipapython.paths' facility.
+COPY ./init/tmpfiles.py /bin/systemd-tmpfiles
```

I like to keep the original file around so we have an easier job debugging.

```suggestion
RUN mv /bin/systemd-tmpfiles /bin/systemd-tmpfiles.orig
COPY ./init/tmpfiles.py /bin/systemd-tmpfiles
```